build.gradle.kts

```diff
@@ -110,15 +110,18 @@ sonar {
                 "dataland-frontend/src/components/resources/frameworkDataSearch/euTaxonomy/" +
                 "EuTaxonomyForNonFinancialsDisplayDataModel.ts," +
                 "dataland-frontend/src/components/resources/frameworkDataSearch/euTaxonomy/configMLDT/" +
-                "configForEutaxonomyFinancialsMLDT.ts",
-
+                "configForEutaxonomyFinancialsMLDT.ts," +
+                "dataland-frontend/src/frameworks/gdv/UploadConfig.ts," +
+                "dataland-frontend/src/frameworks/gdv/ViewConfig.ts",
         )
         property(
             "sonar.exclusions",
             "dataland-backend/src/main/kotlin/" +
                 "org/dataland/datalandbackend/model/enums/eutaxonomy/nonfinancials/Activity.kt," +
                 "dataland-frontend/src/components/general/SlideShow.vue," +
-                "dataland-frontend/src/components/resources/newLandingPage/TheQuotes.vue",
+                "dataland-frontend/src/components/resources/newLandingPage/TheQuotes.vue," +
+            "dataland-backend/src/main/kotlin/org/dataland/datalandbackend/frameworks/**/*",
```

Comment:
Is this intended?

Reply:
Yes. This has been done by @marcelrasbachdfine.  We don't want sonar to analyze our (auto-generated) framework-related Kotlin data classes.  It complains about import statements etc., but we do not want to change that manually because everything in this directory is auto-generated.

Reply:
However, I don't understand why @marcelrasbachdfine has not wrote it like this:

`"dataland-backend/src/main/kotlin/org/dataland/datalandbackend/frameworks/**`

Don't we want everything in there to be ignored?